test/support/utils_invoke.h

```diff
@@ -162,7 +162,7 @@ struct invoke_on_all_policies
     void
     operator()(Op op, T&&... rest)
     {
-        invoke_on_all_host_policies()(op, ::std::forward<T>(rest)...);
+        invoke_on_all_host_policies()(op, rest...);
```

Comment:
What if `TEST_DPCPP_BACKEND_PRESENT` defined in 0?

Reply:
Fixed.